eng/Publishing.props

```diff
@@ -6,9 +6,4 @@
     <PublishingVersion>3</PublishingVersion>
   </PropertyGroup>
 
-  <ItemGroup>
-    <SdkAssetsToPublish Include="$(ArtifactsShippingPackagesDir)*.msi" />
-    <SdkAssetsToPublish Include="$(ArtifactsNonShippingPackagesDir)*.wixpack.zip" />
-  </ItemGroup>
-
```

Comment:
This doesn't do anything. `SdkAssetsToPublish` is not an item group in this repo/Arcade.